HyperPlay-Gaming/hyperplay-desktop-client
# feat(quest):  add balance check and claim error CTA message

## Summary

* Add a warning message when the connected wallet doesn't have enough gas to cover the transaction

![Screenshot 2024-07-24 at 4 07 57 PM](https://github.com/user-attachments/assets/2a9e3900-1ccd-46ca-9cad-e989a101c6b9)

* Add an error message when there is an error claiming the quest

![Screenshot 2024-07-24 at 4 15 07 PM](https://github.com/user-attachments/assets/bbcae5e1-0d51-4503-bbb5-2e3af33a93d3)


### How to test

* Balance check: https://app.qase.io/case/HC-352
* Claim error: https://app.qase.io/case/HC-353

---

Use the following Checklist if you have changed something on the Backend or Frontend:
